packages/flutter/lib/src/services/text_input.dart

```diff
@@ -2273,6 +2299,11 @@ class _PlatformTextInputControl with TextInputControl {
     _channel.invokeMethod<void>('TextInput.show');
   }
 
+  @override
+  void startLiveTextInput() {
+    _channel.invokeMethod<void>('TextInput.startLiveTextInput');
+  }
```

Comment:
What do you think about separating this feature out of TextInput?  Would it be possible/reasonable?

Take a look at this PR where I'm attempting to move Scribble out: https://github.com/flutter/flutter/pull/115296/files

Maybe it doesn't make sense to separate it in this case because 1. TextInputClient is the main concern, and it is unmodified in this PR and 2. Live Text text changes are sent directly to the active input in the engine, so it doesn't make sense to use this feature outside of a text field.

Just thought I'd make sure we thought about it.

Reply:
I think putting it here is ok. Different from Scribble, I think this live text input just like an single operation like `copy` or `paste`. It is possible to move out but I think it is not very very reasonable. So I think it is ok to just adding it here O(∩_∩)O.

Reply:
Alright sounds good, I'm on board with keeping it as-is.